drafts/bitcartel-nu0-tx-format/draft.rst

```diff
@@ -0,0 +1,256 @@
+----------------
+
+As defined in Network Upgrade Mechanism [#zip-0???]_ every network upgrade can be identified by a branch id, a globally-unique non-zero 32-bit identifier.
+
+A format branch id is the branch id when a new transaction format version was introduced on the main chain of a client implementation.  Overwinter requires a transaction to include the format branch id.
+
+Including a format branch id prevents a client on one branch of the network from attempting to parse transactions intended for another branch, in the situation where the transactions share the same format version number but are specified differently.
+
+The combination of transaction version and format branch id, `nVersion || nFormatBranchId` uniquely defines the transaction format, thus enabling parsers to reject transactions from outside the client's chain which cannot be parsed, providing users with replay protection. 
+
+For example, at a block height H the reference implementation of Zcash introduces a new transaction version 3 format as part of the Overwinter network upgrade and assigns a branch id of 0x11111111.  Meanwhile, Zclone at that same block height decides to fork the network upgrade, creating a parallel branch with branch id 0xCCCCCCCC and modifying transaction version 3 format with new fields.  The Zcash parser will reject the Zclone transactions and vice versa, ensuring replay protection, even though both clients create transactions with the version number of 3.
+
+Expiry Height
+-------------
+
+The expiry height field specifies the last block height at which a transaction must be mined into a block, after which the transaction is deemed to have expired and should be removed from the mempool.  A block is invalid if it contains an expired transaction.  More information can be found in the Transaction Expiry ZIP [#zip-???]_
+
+Transaction Validation
+======================
+
+A client which supports the consensus rules identified by the format branch id should accept the transaction for further validation.  If the value of the branch id is zero, this indicates the absence of any network upgrade and that Sprout consensus rules apply.
+
+Overwinter parsers should reject transactions from further processing and validation if the:
+
+- version number is unknown
+- overwinter flag is not set
+- format branch id is not set
+
+Overwinter clients should reject transactions for violating consensus rules if:
+
+- version number is invalid according to current set of active consensus rules
+- format branch id is different from the branch id corresponding to the current set of active consensus rules
+- mis-matched format branch id causes signatures to fail verification under the new Overwinter signature hashing scheme which includes a branch id
+
+Implementation
+==============
+
+Transaction Version
+-------------------
+
+Transaction version remains a positive value.
+
+Code can continue to check the transaction version using comparison operators::
```

Comment:
Clarify that this is not the case inside (de)serialization code, where the pair `(nVersion, nVersionGroupId)` must be checked.